### PR TITLE
test(ui): Convert `organizationAuthList.spec` to ts

### PR DESCRIPTION
**Pull request by billyvg**

### CODE REVIEW

File: static/app/views/settings/organizationAuth/organizationAuthList.spec.tsx [hunk 1]
```diff
@@ -64,14 +66,12 @@ describe('OrganizationAuthList', function () {
     });
 
     it('renders with saml available', function () {
```
Comment (EricHasegawa) [inline static/app/views/settings/organizationAuth/organizationAuthList.spec.tsx:68]: nit: why isn't SAML capitalized (or SSO below)

**Review by EricHasegawa — APPROVED** [review]
Are you improving the tests here in addition to the typescript conversion? Just wanna double check

**Comment by billyvg** [comment]
Yeah the tests with `snapshot` aren't really doing anything so I removed them

File: static/app/views/settings/organizationAuth/organizationAuthList.spec.tsx (hunk 1 above)
Comment (billyvg) [reply static/app/views/settings/organizationAuth/organizationAuthList.spec.tsx:68]: Not sure

**Pull request merged**